extensions/common/context.h

```diff
@@ -134,6 +134,12 @@ struct RequestInfo {
   std::string url_path;
   std::string url_host;
   std::string url_scheme;
+
+  // TCP variables.
+  int64_t tcp_connections_opened = 0;
+  int64_t tcp_connections_closed = 0;
```

Comment:
We don't need the variable to keep track of open/close, right? It is also +1 for both metrics.

Reply:
yes... but I think if we do it periodically, we need to keep these variables...